lark/lark.py

```diff
@@ -23,41 +19,51 @@
 except ImportError:
     regex = None
 
+
+###{standalone
```

Comment:
@erezsh This can happen in another PR, but we might want to consider removing type annotations like we are removing docstrings from the standalone parser.

Reply:
I agree that we should, if we can.